README.md

```diff
@@ -26,39 +26,57 @@ The script also enables backtraces by setting `RUST_BACKTRACE=1` env variable.
 
 ## Starting the maker and taker daemon
 
+A working example of starting both daemons with all the required command-line parameters can be found
+[here](https://github.com/itchysats/itchysats/blob/collab-settlement-test/start_all.sh#L8)
```

Comment:
Why is this not pointing to `master`?

Reply:
lol, I forgot that my links are generated based on the branch I was on, should have jumped to master first :)
thanks for spotting, otherwise the link would get broken with bors merge of the other PR